examples/user_interfaces/svg_tooltip.py

```diff
@@ -81,16 +62,14 @@
 tree, xmlid = ET.XMLID(f.getvalue())
 tree.set('onload', 'init(evt)')
 
-# Hide the tooltips
-for i, t in enumerate(ax.texts):
-    el = xmlid['tooltip_%d' % i]
-    el.set('visibility', 'hidden')
-
-# Assign onmouseover and onmouseout callbacks to patches.
-for i, t in enumerate(ax.patches):
-    el = xmlid['patch_%d' % i]
-    el.set('onmouseover', "ShowTooltip(this)")
-    el.set('onmouseout', "HideTooltip(this)")
+for i, y in enumerate(shapes):
```

Comment:
Hang on a sec.  It appears that `tooltip = xmlid['mytooltip_%d' % i]` throws a TypeError if y is not used.

Disagreed.